rust/nasl-syntax/src/keyword_extension.rs

```diff
@@ -72,10 +66,248 @@ impl<'a> Lexer<'a> {
             }
         };
         Ok((
-            PrefixState::Break,
+            PrefixState::Break(Category::Semicolon),
             Statement::If(Box::new(condition), Box::new(body), r#else.map(Box::new)),
         ))
     }
+
+    fn paren_base(&mut self) -> Result<(), SyntaxError> {
+        let token = self
+            .token()
+            .ok_or_else(|| unexpected_end!("expected paren."))?;
+        if token.category() != Category::LeftParen {
+            Err(unexpected_token!(token))
+        } else {
+            Ok(())
+        }
+    }
+
+    fn parse_exit(&mut self) -> Result<(PrefixState, Statement), SyntaxError> {
```

Comment:
parse_exit and parse_include are similar, should use same basis instead of repetition